src/services/GameClock.inl.hxx

```diff
@@ -1,15 +1,43 @@
-#include "GameClock.hxx"
+template <typename DelayType, typename PeriodType>
+GameClock::ClockTaskHndl GameClock::addRealTimeClockTask(ClockCbk cbk, DelayType delay, PeriodType period)
```

Comment:
The issue with this design is that an animation will not be able to remove itself. We need a better design where the callback provides its intention (e.g. reschedule me in 1min)

Reply:
Now callback has a boolean return value which in case true stopping task rescheduling.
This implementation should support animations as planned.